src/Connection.php

```diff
@@ -304,28 +319,32 @@ public function cursorWithOptions(string $query, array $bindings, array $options
     }
 
     /**
-     * @inheritDoc
+     * {@inheritDoc}
+     * @param array<array-key, mixed> $bindings
      */
     public function statement($query, $bindings = []): bool
     {
         // is SELECT query
         if (0 === stripos(ltrim($query), 'select')) {
-            return $this->select($query, $bindings) !== null;
+            $this->select($query, $bindings);
+            return true;
```

Comment:
This fix is unrelated to phpstan. Please submit it in a separate PR.

Reply:
I have read the code to make sure that the change is correct, but it should not be false here, and PHPStan always warns as true, so I am making this change. Should I still add it to ignore?

Reply:
it means
- `$this->select()` method has language level return-type non-nullable `array`, `!== null` is always true.
- `$this->affectingStatement()` method has language level return-type non-nullable `int` , `!== null` is always true.

Reply:
Understood. 
It would have been nice if it was issued as a fix so it was easier to see what was going on.
Also would have made the review process slightly less painful for me.